src/server/auth/server/api_server.go

```diff
@@ -1834,35 +1867,47 @@ func lenientCanonicalizeSubjects(ctx context.Context, subjects []string) ([]stri
 
 // lenientCanonicalizeSubject is like 'canonicalizeSubject', except that if
 // 'subject' has no prefix, they are assumed to be a GitHub user.
-func lenientCanonicalizeSubject(ctx context.Context, subject string) (string, error) {
+func (a *apiServer) lenientCanonicalizeSubject(ctx context.Context, subject string) (string, error) {
```

Comment:
Would it be dangerous to roll the behavior of `lenientCanonicalizeSubject` into `CanonicalizeSubject` it seems a little lame to have both, and it seems like you're mostly using the lenient version here. Are there cases where you need to be sure it actually has its prefix?

Reply:
Done. For the record, my preference would actually be to get rid of `lenientCanonicalizeSubject` completely and not assume any prefixes, but the dash isn't ready for that yet, and the way the code is laid out doesn't really affect that.